todocore/src/main/java/com/knighttodo/todocore/rest/DayResource.java

```diff
@@ -147,4 +148,23 @@ public void deleteDay(@PathVariable UUID dayId) {
             throw new DayCanNotBeDeletedException("Day can't be deleted.", ex);
         }
     }
+
+    @GetMapping("/date")
+    @ResponseStatus(HttpStatus.OK)
+    @ApiOperation(value = "Find all Days by Date", response = DayResponseDto.class, responseContainer = "List")
+    @ApiParam(value = "Find day by date", example = "2023-02-06", required = true, readOnly = true)
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Days found"),
+            @ApiResponse(code = 400, message = "Invalid operation"),
+            @ApiResponse(code = 403, message = "Operation forbidden"),
+            @ApiResponse(code = 500, message = "Unexpected error")
+    })
+    public DayResponseDto findDaysByDate(@RequestParam(name = "date") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) LocalDate date) {
+        try {
+            return dayRestMapper.toDayResponseDto(dayService.findDayByDate(date));
+        } catch (RuntimeException ex) {
+            log.error("Days can't be found by date: " +date.toString(), ex);
+            throw new FindAllDaysException("Days can't be found.", ex);
```

Comment:
Create a separate Exception for this request please

Reply:
Done!